src/layer/vector/CircleMarker.js

```diff
@@ -33,9 +33,13 @@ export var CircleMarker = Path.extend({
 	// @method setLatLng(latLng: LatLng): this
 	// Sets the position of a circle marker to a new location.
 	setLatLng: function (latlng) {
+		var oldLatLng = this._latlng;
 		this._latlng = toLatLng(latlng);
 		this.redraw();
-		return this.fire('move', {latlng: this._latlng});
+
+		// @event move: Event
+		// Fired when the marker is moved via [`setLatLng`](#marker-setlatlng) or by [dragging](#marker-dragging). Old and new coordinates are included in event arguments as `oldLatLng`, `latlng`.
```

Comment:
This should be `#circlemarker-setlatlng` instead of `#marker-setlatlng`. Also, `CircleMarker`s do not have a `draggable` option as `Marker`s do.

Reply:
Oh I did't see that sorry should be fixed now :-)